### PR TITLE
Add input option for Model Name in GoogleVertexAIEmbedding

### PR DESCRIPTION
**Pull request by Yongtae723**
Adaptation of [multi-lingual support of Vertex AI Embedding](https://github.com/Yongtae723/Flowise/pull/new/fix/add_new_model_embedding_vertexai), which is published Google Could Next on last week! 🎉

### CODE REVIEW

**Review by HenryHengZJ — APPROVED** [review]
thank you @Yongtae723 !

File: packages/components/nodes/embeddings/GoogleVertexAIEmbedding/GoogleVertexAIEmbedding.ts [hunk 1]
```diff
@@ -33,11 +33,35 @@ class GoogleVertexAIEmbedding_Embeddings implements INode {
...
+                        name: 'textembedding-gecko@latest'
+                    },
+                    {
+                        label: 'textembedding-gecko-multilingual@latest',
+                        name: 'textembedding-gecko-multilingual@latest'
+                    }
+                ],
+                default: 'textembedding-gecko@001',
+                optional: true,
+                additionalParams: true
```
Comment (HenryHengZJ) [inline packages/components/nodes/embeddings/GoogleVertexAIEmbedding/GoogleVertexAIEmbedding.ts:57]: maybe we don't have to use `additionalParams` here

File: packages/components/nodes/embeddings/GoogleVertexAIEmbedding/GoogleVertexAIEmbedding.ts [hunk 2]
```diff
@@ -58,7 +82,9 @@ class GoogleVertexAIEmbedding_Embeddings implements INode {
 
             if (projectID) authOptions.projectId = projectID
         }
-        const obj: GoogleVertexAIEmbeddingsParams = {}
+        const obj: GoogleVertexAIEmbeddingsParams = {
+            model: modelName
```
Comment (HenryHengZJ) [inline packages/components/nodes/embeddings/GoogleVertexAIEmbedding/GoogleVertexAIEmbedding.ts:86]: if `modelName` is undefined, is it still running fine? maybe a better way is to check:
```
if (modelName) obj.model = modelName
```

File: packages/components/nodes/embeddings/GoogleVertexAIEmbedding/GoogleVertexAIEmbedding.ts (hunk 1 above)
Comment (Yongtae723) [reply packages/components/nodes/embeddings/GoogleVertexAIEmbedding/GoogleVertexAIEmbedding.ts:57]: Got it！

File: packages/components/nodes/embeddings/GoogleVertexAIEmbedding/GoogleVertexAIEmbedding.ts (hunk 2 above)
Comment (Yongtae723) [reply packages/components/nodes/embeddings/GoogleVertexAIEmbedding/GoogleVertexAIEmbedding.ts:86]: Oh yeah!

you are correct!
thank you for pointing out!

**Comment by Yongtae723** [comment]
@HenryHengZJ thank you for your comment!

I edited as your suggestion!

**Pull request merged**